docs/strategies/ios.md

```diff
@@ -0,0 +1,12 @@
+# iOS Analysis
```

Comment:
I'm conflicted on the naming here. Maybe `swift/objective-c` would be better here? In general we need to rethink strategy doc grouping, but that's out of scope for this PR in particular